trulens_eval/trulens_eval/feedback/provider/hugs.py

```diff
@@ -25,6 +25,7 @@
 HUGS_NLI_API_URL = "https://api-inference.huggingface.co/models/ynie/roberta-large-snli_mnli_fever_anli_R1_R2_R3-nli"
 HUGS_DOCNLI_API_URL = "https://api-inference.huggingface.co/models/MoritzLaurer/DeBERTa-v3-base-mnli-fever-docnli-ling-2c"
 HUGS_PII_DETECTION_API_URL = "https://api-inference.huggingface.co/models/bigcode/starpii"
+HUGS_CONTEXT_RELEVANCE_API_URL = "https://api-inference.huggingface.co/models/truera/context_relevance"
```

Comment:
are there limitations to this api? (rate, etc.)

Are there docs to add on how it was trained/benchmarked, and why someone should use it? Or is that coming with a later release